compact_bilinear_pooling/__init__.py

```diff
@@ -3,11 +3,23 @@
 import torch.nn as nn
 from torch.autograd import Function
 
+try:
+    from torch import irfft
+    from torch import rfft
+except ImportError:
+    def rfft(x, d):
+        t = torch.fft.fft(x, dim=(-d))
```

Comment:
Thanks for the PR. Why using `torch.fft.fft` instead of `torch.fft.rfft`? The original code worked with real numbers, and when I tested it, I found that rfft was faster and used less memory than fft.